templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -5,13 +5,33 @@ param tags object = {}
 param containerAppsEnvironmentName string
 param containerName string = 'main'
 param containerRegistryName string
+
+@description('Minimum number of replicas to run')
+param containerMinReplicas int = 1
+@description('Maximum number of replicas to run')
```

Comment:
Similar feedback, perhaps we should use `@minValue` here to enforce this is always 1 or more?

Reply:
done